.appveyor.yml

```diff
@@ -39,8 +39,9 @@ test_script:
   - yarn type-check
   # FIXME: Exclude Appveyor from running `perf` smoketest until we fix the flake.
   #     https://github.com/GoogleChrome/lighthouse/issues/5053
-  # - yarn smoke
-  - yarn smoke a11y pwa pwa2 pwa3 dbw redirects seo offline byte metrics
+  # - yarn smoke -j=1 --retries=3
+  # run smoke tests serially, retrying up to three times to prevent flakes.
+  - yarn smoke -j=1 --retries=3 a11y pwa pwa2 pwa3 dbw redirects seo offline byte metrics
```

Comment:
> unrelated to this pr but...
> 
> this is also missing oopif, lantern, errors.

ha, well that's not great

> would be better if we explicitly skipped `perf` if `process.env.APPVEYOR`.

I'd like to move most of this away from deep environment variable checks. It's really difficult to think through execution (e.g. what's running on appveyor) when there's a surprise three files deep that alters execution instead of doing it at the original callsite. It's not always avoidable, but we should avoid it where we can.

All that said, this is a valuable use case, so I'll try to think of something else or fall back to a `process.env.APPVEYOR` check if we have to :)